### DISCUSSION TITLE
Problem with glasses

### DISCUSSION THREAD

**Discussion by CN1o2** [BEEmod/BEE2-items#3913] [Tech Support / Q&A] [answered]
![Beemod 5](https://user-images.githubusercontent.com/88097966/132104800-b96523b7-5bad-4750-9566-c94ab1cf8189.png)

Steps to reproduce the problem:
I start the map a second time and then this happens

**Comment by slambo311** [comment]
This should be in issues, not discussions.
Also, you gotta be more specific as to what you did. You need to post your OS (Windows, Mac, Linux), BEEmod version, BEEmod package version, BEEmod style (Clean, Overgrown, BTS), BEEmod skybox (Clean, Sphere Exterior, Sphere Interior), items used in the test, all that stuff.

**Comment by Konclan** [comment, marked as answer]
This happens when you load the map and go into play mode without building the map, the cubemaps default to editor default instead of being blank like they should. This isn't a BEEmod issue.
